roles/capsule_certs_generate/README.md

```diff
@@ -0,0 +1,31 @@
+redhat.satellite-operations.capsule_certs_generate
+==================================================
+
+Generates a certificate bundle for Satellite scenario
+
+Role Variables
+--------------
+
+Required:
+
+- `capsule_certs_generate_fqdn`: Target FQDN to generate a certificate bundle for
```

Comment:
wonder if the vars should start with `satellite_capsule` for cleaner namespacing?

Reply:
Solid question.